setup_script.sh

```diff
@@ -34,6 +34,78 @@ if ! command -v "node" &>/dev/null; then
     exit
 fi
 
+###
+# check for the existence of required files in the Distro and camicroscope repository
+###
+echo "Checking for required files"
+
+if [[ ! -d ../Distro/db ]]
```

Comment:
For this, it _should_ be sufficient to check that whichever mongo uri is being used is accessible. Distro/db is just a hacky way to ensure persistence/fs backups of the mongo data.

Reply:
I have removed the lines as there is a check for mongo uri in the setup script and committed the changes: https://github.com/camicroscope/Caracal/pull/101/commits/05bd6cc58b3ee5b9f2c6fda97eb1897ebbc6a79f